CONTRIBUTORS.md

```diff
@@ -0,0 +1,14 @@
+# Contributors
+
+Thank you to all the people who have already contributed to git-point!
```

Comment:
> git-point

Let's rename to GitPoint?

Reply:
It looks like an actual repository name, that PR author written here...